content/docs/command-reference/remote/modify.md

```diff
@@ -1202,16 +1202,27 @@ by HDFS. Read more about by expanding the WebHDFS section in
   1. `user` parameter set with this command (found in `.dvc/config`);
   2. User defined in the URL (e.g. `webdavs://user@example.com/endpoint/path`)
 
-- `password` - password for WebDAV server, can be empty in case of using `token`
-  authentication.
+- `custom_auth_header` - HTTP header field name to use for authentication. Value
+  is set via `password`.
+
+  ```cli
+  $ dvc remote modify --local myremote \
+                      custom_auth_header 'My-Header'
+  ```
+
+- `password` - password for WebDAV server, combined either with `user` or
+  `custom_auth_header`. Leave empty for `token` authentication.
 
   ```cli
   $ dvc remote modify --local myremote password mypassword
   ```
 
-> Note that `user/password` and `token` authentication are incompatible. You
-> should authenticate against your WebDAV remote by either `user/password` or
-> `token`.
+  <admon type="info">
+
+  Auth based on `user` or `custom_auth_header` (with `password`) is incompatible
+  with `token` auth.
+
+  </admon>
```

Comment:
TBH I'm not sure we need this note. I'm guessing `dvc push` will show a specific error if both are set? Merging for now anyway. Thanks @frederik-sandfort and @skshetry 🙂